testutil/testnode/full_node_test.go

```diff
@@ -106,3 +107,38 @@ func (s *IntegrationTestSuite) Test_FillBlock() {
 		require.Equal(uint64(squareSize), b.Block.OriginalSquareSize)
 	}
 }
+
+func (s *IntegrationTestSuite) Test_FillBlock_InvalidSquareSizeError() {
+	tests := []struct {
+		name        string
+		squareSize  int
+		expectedErr error
+	}{
+		{
+			name:        "when squareSize less than 2",
+			squareSize:  0,
+			expectedErr: fmt.Errorf("unsupported squareSize: 0"),
+		},
+		{
+			name:        "when squareSize equal to 2",
+			squareSize:  2,
+			expectedErr: fmt.Errorf("unsupported squareSize: 2"),
+		},
```

Comment:
If the other suggestion is applied, this test case can be removed

```suggestion
```

Reply:
Removed this case